work/v1alpha1/types.go

```diff
@@ -0,0 +1,158 @@
+}
+
+// PlaceManifestWorkSpec defines the desired state of PlaceManifestWork
+type PlaceManifestWorkSpec struct {
+	// ManifestWorkSpec is the ManifestWorkSpec that will be used to generate a per-cluster ManifestWork
+	ManifestWorkTemplate work.ManifestWorkSpec `json:"manifestWorkTemplate"`
+
+	// PacementRef is the name of the Placement resource, from which a PlacementDecision will be found and used
+	// to distribute the ManifestWork
+	PlacementRef LocalPlacementReference `json:"placementRef,omitempty"`
+}
+
+// PlaceManifestWorkStatus defines the observed state of PlaceManifestWork
+type PlaceManifestWorkStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Conditions contains the different condition statuses for distrbution of ManifestWork resources
+	// Valid condition types are:
+	// 1. AppliedManifestWorks represents ManifestWorks have been distributed as per placement All, Partial, None, Problem
+	// 2. PlacementRefValid
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+
+	// ManifestWorkDelivery tracks each ManifestWork that was created,
+	// including that it was applied and its overall status
+	PlacedManifestWork []PlacedManifestWork `json:"placedManifestWork,omitempty"`
+
+	// Summary totals of resulting ManifestWorks
+	PlacedManifestWorkSummary []PlacedManifestWorkSummary `json:"summary"`
+}
+
+// localPlacementReference is the name of a Placement resource in current namespace
+type LocalPlacementReference struct {
+	// Name of the Placement resource in the current namespace
+	Name string `json:"name"`
+}
+
+// PlaceManifestWorkSummary provides reference counts of all ManifestWorks that are associated with a
+// given PlaceManifestWork resource, for their respective states
+type PlacedManifestWorkSummary struct {
+	// Total number of ManifestWorks managed by the PlaceManifestWork
+	Total int `json:"total"`
+	// TODO: Progressing is the number of ManifestWorks with condition Progressing: true
+	Progressing int `json:"progressing"`
+	// Available is the number of ManifestWorks with condition Available: true
+	Available int `json:"available"`
+	// TODO: Degraded is the number of ManifestWorks with condition Degraded: true
+	Degraded int `json:"degraded"`
+	// Applied is the number of ManifestWorks with condition Applied: true
+	Applied int `json:"Applied"`
+}
+
+type ManifestWorkStatus string
+
+// PlacedManifestWork is a minimal status of the ManifestWork created for each targeted clusters, 0..n
+type PlacedManifestWork struct {
+
+	// Work is a name & namespace reference to the actual ManifestWork resource
+	Work ManifestWorkReference `json:"workRef"`
+
+	// Condition stores the last known status of the ManifestWork
+	// The ManifestWork generation can be included as well as the last known condition
+	// +optional
+	Condition metav1.Condition `json:"condition,omitempty"`
```

Comment:
Conditions []metav1.Condition ?

Reply:
I only planned to represent one condition that replaces applied, in other words keep the latest applicable condition. Storing multiple conditions by thousands of clusters might likely be a problem as @serngawy suggests.

Reply:
Removing.